.github/workflows/conformance-aks-v1.11.yaml

```diff
@@ -72,6 +72,7 @@ env:
   k8s_version: 1.23
   # renovate: datasource=github-releases depName=cilium/cilium-cli
   cilium_cli_version: v0.13.2
+  cilium_cli_ci_version:
```

Comment:
Do we really need to set this in every workflow? Can't we just keep `with.relase-version` when using the action below and rely on the `with.ci-version` default value being empty?